### PR TITLE
Added previous publication for PGS-Catalog citation and updated acknowledgements

### PR DESCRIPTION
**Pull request by fyvon**
( + update of rest/info examples)

### CODE REVIEW

File: rest_api/static/rest_api/openapi/openapi-schema.yml [hunk 1]
```diff
@@ -2404,32 +2432,34 @@ paths:
...
-                          example: 'Lambert et al'
-                        journal:
-                          type: string
-                          description: "Name of the journal"
-                          example: 'Nature Genetics'
-                        year:
-                          type: number
-                          description: "Year of publication"
-                          example: 2021
+                      $ref: '#/components/schemas/PGSCatalogPublication'
```
Comment (ens-lgil) [inline rest_api/static/rest_api/openapi/openapi-schema.yml:2435]: Do we need more than 1 example for that ? I though we would only display the latest publication for the REST API

File: catalog/templates/catalog/docs/about.html [hunk 2]
```diff
@@ -90,12 +90,19 @@ <h4 id="citation">Citation</h4>
         under the supervision of <a href="https://www.inouyelab.org/home/people">Michael Inouye</a> (<a href="https://sysgenresearch.org/">University of Cambridge & Baker Institute</a>)
         in collaboration with <a href="{{ hdr_uk_cam_url }}">Health Data Research - UK</a> (Laurent Gil)
         and the <a href="{{ ebi_url }}/gwas/">EBI Samples, Phenotypes and Ontologies team / NHGRI-EBI GWAS Catalog</a> (Helen Parkinson, Aoife McMahon, Laura Harris).</p>
-    <p>The Catalog is under active development, and we continue to add new features and curate new data. If you use the Catalog or Calculator in your research we ask that you cite our recent publication:
+    <p>The Catalog is under active development, and we continue to add new features and curate new data. If you use the Catalog or Calculator in your research we ask that you cite our below flagship publications:
     <div class="clearfix mb-3">
       <div class="highlight_content">
```
Comment (ens-lgil) [inline catalog/templates/catalog/docs/about.html:95]: Wondering, as the code is the same as in the FAQ, if we should create a small template for the citation and `include` it for the about and FAQ pages

File: rest_api/static/rest_api/openapi/openapi-schema.yml (hunk 1 above)
Comment (fyvon) [reply rest_api/static/rest_api/openapi/openapi-schema.yml:2435]: Initially I found weird to have 2 times the same publication, but I can roll it back to previous format

File: catalog/templates/catalog/docs/about.html (hunk 2 above)
Comment (fyvon) [reply catalog/templates/catalog/docs/about.html:95]: Yes that makes sense, I will add that.

File: rest_api/static/rest_api/openapi/openapi-schema.yml (hunk 1 above)
Comment (ens-lgil) [reply rest_api/static/rest_api/openapi/openapi-schema.yml:2435]: Yes, I don't think we need all the examples there. Just one in an array should be enough

**Review by ens-lgil — APPROVED** [review]

**Pull request merged**